src/decoders/citrix_ctx1_decoder.rs

```
@@ -37,7 +37,7 @@ impl Crack for Decoder<CitrixCTX1Decoder> {
         let mut results = CrackResult::new(self, text.to_string());
 
         if decoded_text.is_none() {
-            debug!("Failed to decode citrix_ctx1 because the length is not a multiple of 4");
+            debug!("Failed to decode citrix_ctx1");
```

Anyway we can propagate an error here or something? Why couldn't it decode?